frontend/src/metabase/admin/settings/components/CloudPanel/MigrationInProgress.tsx

```diff
@@ -0,0 +1,113 @@
+import { c, t } from "ttag";
+
+import { useCancelCloudMigrationMutation } from "metabase/api";
+import { useSetting } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { useToggle } from "metabase/hooks/use-toggle";
+import { useDispatch } from "metabase/lib/redux";
+import { addUndo } from "metabase/redux/undo";
+import { Flex, Text, List, Button, Box, Modal, Progress } from "metabase/ui";
+
+import { MigrationCard } from "./CloudPanel.styled";
+import type { InProgressCloudMigration, InProgressStates } from "./utils";
+import { getCheckoutUrl } from "./utils";
+
+interface MigrationInProgressProps {
+  migration: InProgressCloudMigration;
+}
+
+const progressMessage: Record<InProgressStates, string> = {
+  init: t`Talking to Metabase Cloud...`,
+  setup: t`Talking to Metabase Cloud...`,
```

Comment:
Should we use the `applicationName` variable here?

Reply:
Nope, this is a harbormaster / separate service.